Small Russian translation + added new untranslatable strings

iMiKED opened the pull request:
Small Russian translation + added new untranslatable strings

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master'

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master'

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master'

iMiKED committed:
Merge branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
OneHandMode Russian translation

OneHandMode Russian translation

iMiKED committed:
deleted .gitignore

deleted .gitignore

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Carrier Label Russian Translation

Carrier Label Russian Translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

pnikolakakis committed:
greek update add "Disable Quicks settings" transl (#65)

Damn.. you where fast :p

dragonpt committed:
[STRINGS][CLEAN]fw base: Remove depricate string

After a recent improvement on this feature, this is no longer used, remove...

dragonpt committed:
Merge branch 'master' of github.com:MediatekAndroidDevelopers/mados_custom_strings

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Disable Quick Settings Russian Translation

Disable Quick Settings Russian Translation

iMiKED committed:
Disable Quick Settings Russian Translation

Disable Quick Settings Russian Translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
OmniJaws Russian Translation

OmniJaws Russian Translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge branch 'refs/heads/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Powermenu Russian Translation

Powermenu Russian Translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
On-The-Go Russian Translation

On-The-Go Russian Translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Idle stats + Supported Gestures fix for Russian translate

Idle stats + Supported Gestures fix for Russian translate

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge branch 'Russian-translation' of https://github.com/iMiKED/mados_custom_strings into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Camera Russian translate

Camera Russian translate

iMiKED committed:
Delete gitignore

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Lockscreen retranslation to Russian

Lockscreen retranslation to Russian

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge branch 'Russian-translation' of https://github.com/iMiKED/mados_custom_strings into Russian-translation

iMiKED committed:
Russian retranslation

Russian retranslation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Russian retranslation + WiFi translation

Russian retranslation + WiFi translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge branch 'Russian-translation' of https://github.com/iMiKED/mados_custom_strings into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Russian translation minor corrections

Russian translation minor corrections

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Lockscreen battery info Russian translation

Lockscreen battery info Russian translation

iMiKED committed:
Merge branch 'Russian-translation' of https://github.com/iMiKED/mados_custom_strings into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
More hardware keys Russian translation

More hardware keys Russian translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge branch 'Russian-translation' of https://github.com/iMiKED/mados_custom_strings into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'origin/Russian-translation' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Color engine+buttons Russian translation

Color engine+buttons Russian translation

iMiKED committed:
Color engine+buttons Russian translation

Color engine+buttons Russian translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Small Russian translations

Small Russian translations

fire855 commented:
Hey, why are you adding >untranslatable< strings in the translation file? That's not needed and makes future changes on these values harder. Please remove them again.

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Merge remote-tracking branch 'refs/remotes/MediatekAndroidDevelopers/master' into Russian-translation

iMiKED committed:
Untranslatable removed

Untranslatable strings was added to match original files number of
strings. They never was translated. Removed them again from Russian
translations.

iMiKED commented:
@fire855 , please, check again. Removed untranslatable strings.
